include/ipfixprobe/packet.hpp

```diff
@@ -59,7 +59,10 @@ struct Packet : public Record {
    uint8_t     ip_flags;
    ipaddr_t    src_ip;
    ipaddr_t    dst_ip;
-   uint32_t    vlan_id;
+   uint16_t    vlan_id;
+   uint32_t    frag_id;
+   uint16_t    frag_off;
```

Comment:
For an IPv6 packet without fragmentation header, `frag_off` and `frag_id` values are undefined as you don't clear them in the constructor.